tdrs-frontend/src/reducers/auth.js

```diff
@@ -32,7 +32,7 @@ const auth = (state = initialState, action) => {
         ...state,
         loading: false,
         authenticated: true,
-        user,
+        user: { ...state.user, ...user },
```

Comment:
This makes sure that user information is kept and new user information can be added up access request.